config/default/user.role.contributor.yml

```diff
@@ -0,0 +1,71 @@
+uuid: caa42067-aba2-4d13-84e5-5fb3e7e78d54
```

Comment:
Role config might have to be split out into each site as each site may have different modules installed, no? This config might be better set on install and then ignored.

Reply:
yeah. each site will probably have its own config. but this file at least sets the initial perms even if we ignore it. i found that with config ignore, the config will get imported but then it wont recieve changes.